datafusion/core/src/physical_plan/joins/nested_loop_join.rs

```diff
@@ -312,11 +296,11 @@ struct NestedLoopJoinStream {
     filter: Option<JoinFilter>,
     /// type of the join
     join_type: JoinType,
-    /// future for data from left side
-    left_fut: OnceFut<JoinLeftData>,
-    /// right
-    right: SendableRecordBatchStream,
-    /// There is nothing to process anymore and left side is processed in case of left/left semi/left anti/full join
+    // the outer table data of the nested loop join
```

Comment:
can we add more comments to indicate which side belonged to the outer table for the different `join type`.

Reply:
Added the comments in the description of `NestedLoopJoinExec`.